docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,228 @@
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+Next, back in the `static void Main(string[] args)` method, add a call `ShowBot` as the last line of the method, passing in the message argument value.
+
+```csharp
+ShowBot(messageArg.Value);
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations to run our application:
+
+```csharp
+dotnet run
+dotnet run -- -h
+dotnet run -- -v
+dotnet run -- "Hello from the bot"
+```
+
+All arguments after the `--` delimiter are passed to our application.
+
+## Setup the global tool
+
+Before you can pack and distribute the application as a Global Tool, you need to modify the project file. Open the `botsay.csproj` file and add three new XML nodes to the `<Project><PropertyGroup>` node:
+
+- \<PackAsTool>  
+Indicates that the application will be compiled for install as a Global Tool.
+
+- \<ToolCommandName>  
+An alternitive name for the tool. Essentially an alias that you can use to call the tool.
+
+- \<PackageOutputPath>  
+Where the NuGet package will be produced. The NuGet package is what the .NET Core CLI Global Tools uses to install your tool.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <PackAsTool>true</PackAsTool>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+    <ToolCommandName>botsay</ToolCommandName>
+
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
+  </ItemGroup>
+
+</Project>
+```
+
+Next, create a NuGet package for your application.
+
+```console
+dotnet pack -c Release
+```
+
+The `botsay.1.0.0.nupkg` file will be created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which should be the `./nupkg` folder.
+
+Now that you have a package, install the tool from that package: 
+
+```console
+dotnet tool install -g --add-source ./nupkg botsay`
+```
+
+The `--add-source` parameter tells .NET Core to use the `./nupkg` folder (our output folder) as a feed source for NuGet packages. For more information about installing Global Tools, see [.NET Core Global Tools overview][global-tool-info].
```

Comment:
as an **additional** feed source

Reply:
maybe we need to mention if the version is lower than 1.0 (which is likely when user start to try it.), say 0.1. You need to use add extra `--version 0.1`